### PR TITLE
README: Document the metadata options.

### PR DESCRIPTION
**Pull request by cfoch**
- sound-file
- sound-files
- loop
- fade-in
- fade-out
- volume
- pitch
- delay
- overlap-behavior

https://phabricator.endlessm.com/T25071

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
...
+
+- **`sound-file`**: Indicates the path to the sound file to be played.
+- **`sound-files`**: It's an array of paths, and indicates that one of these sounds should be picked up randomly to be played. If like in this example, `sound-file` and `sound-files` are set, then all the specified paths will be considered, in other words, for this example, one sound among "sound0.wav", "sound1.wav", "sound2.wav" and "sound3.wav" would be played.
+- **`loop`**: If set to `true` the sound will be played again when it finishes. *Defaults to `false`*.
+- **`fade-in`**: Indicates the time duration in which the volume of the sound should fade in from the start. The used unit is milliseconds. *Defaults to 1000 only if `loop` is set to `true`.*
+- **`fade-out`**: Indicates the time duration in which the volume of the sound should fade out after the sound is stopped. *The used unit is milliseconds. Defaults to 1000 only if `loop` is set to `true`.*
+- **`volume`**: Indicates the volume level the sound you play at. *Defaults to 1.0 (the "normal" volume).*
+- **`pitch`**: Sets the sound pitch while keeping the original tempo (speed). *Defaults to 1.0 (the "normal" pitch).
```
Comment (dnohales) [inline README.md]: Forgot to close italics.

File: README.md [hunk 2]
```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
...
+- **`rate`**: Sets the tempo and pitch. *Defaults to 1.0 (the "normal" pitch*.
```
Comment (dnohales) [inline README.md]: Forgot to close parenthesis, also, "the normal pitch"? wouldn't be "the normal rate/tempo and pitch"?

File: README.md [hunk 3]
```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
...
+        "delay": 10000,
+        "overlap-behavior": "restart"
+    }
+```
+This example shows all the options that the metadata file accepts.
+
+- **`sound-file`**: Indicates the path to the sound file to be played.
+- **`sound-files`**: It's an array of paths, and indicates that one of these sounds should be picked up randomly to be played. If like in this example, `sound-file` and `sound-files` are set, then all the specified paths will be considered, in other words, for this example, one sound among "sound0.wav", "sound1.wav", "sound2.wav" and "sound3.wav" would be played.
+- **`loop`**: If set to `true` the sound will be played again when it finishes. *Defaults to `false`*.
+- **`fade-in`**: Indicates the time duration in which the volume of the sound should fade in from the start. The used unit is milliseconds. *Defaults to 1000 only if `loop` is set to `true`.*
```
Comment (dnohales) [inline README.md]: Punctuation after finishing italics

File: README.md [hunk 4]
```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
...
+- **`fade-out`**: Indicates the time duration in which the volume of the sound should fade out after the sound is stopped. *The used unit is milliseconds. Defaults to 1000 only if `loop` is set to `true`.*
```
Comment (dnohales) [inline README.md]: Punctuation after finishing italics

File: README.md [hunk 5]
```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
...
+- **`volume`**: Indicates the volume level the sound you play at. *Defaults to 1.0 (the "normal" volume).*
```
Comment (dnohales) [inline README.md]: Punctuation after finishing italics

File: README.md [hunk 6]
```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
...
+- **`pitch`**: Sets the sound pitch while keeping the original tempo (speed). *Defaults to 1.0 (the "normal" pitch).
+- **`rate`**: Sets the tempo and pitch. *Defaults to 1.0 (the "normal" pitch*.
+- **`delay`**: Sets the duration time that should be delayed before the sound starts to be played. *The used unit is milliseconds. Defaults to 0*.
```
Comment (dnohales) [inline README.md]:
```
- **`delay`**: The duration in milliseconds that should be delayed before the sound starts. *Defaults to 0*.
```

File: README.md [hunk 7]
```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
...
+- **`overlap-behavior`**: Indicates the behavior of the sound when the same sound is requested to be played while the other is also playing. The available options are: `"overlap"`, `"ignore"` and `"restart"`. If `"overlap"` is set, then if the same sound is played twice or more times simultaneously, all these sounds will overlap between them. If `"ignore"` is set, if the target sound is already playing and an application requests to play this sound, this request will be ignored: this means that **an only one** instance of the sound will be played. If `"restart"` is set, then if the target sound is already playing and an application requests to play this sound, the sound will be restarted: this (also) means that **an only one** instance of the sound will be playing. *Defaults to `"overlap"`*.
```
Comment (dnohales) [inline README.md]: an only one -> only one

File: README.md [hunk 8]
```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
...
+## A full example
+```
+{
+    "example/sound/1": {
+        "sound-file": "sound0.wav",
+        "sound-files": [
+            "sound1.wav",
+            "sound2.wav",
+            "sound3.wav"
+		]
```
[the remark this answers is not in the document]
Comment (cfoch) [reply README.md]: and I forgot a "," :-1:

**Pull request merged**